www/js/app.js

```diff
@@ -104,11 +105,9 @@ serviceApp.factory('Users',function($http, $q, $stateParams) {
     var deleteUser = function(id){
       for(i=0; i<users.length; i++){
         if(users[i].id == id){
-          //delete users[i];
           users.splice(i,1);
```

Comment:
Same here: pourquoi tu continues à itérer sur le tableau dès que tu as trouvé ton élément ?